pkg/controller/import-controller.go

```diff
@@ -425,13 +434,42 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 			r.recorder.Event(pvc, corev1.EventTypeNormal, ImportSucceededPVC, "Import Successful")
 			log.V(1).Info("Completed successfully, deleting POD", "pod.Name", pod.Name)
 		}
+		if getSource(pvc) == SourceVDDK {
+			r.saveVddkAnnotations(pvc, pod, log)
```

Comment:
Done.

Reply:
Undone, in favor of removing the embedded call to Update. Now the only real reason it can fail is if it can't read the info fields from the termination message. The termination message is logged enough to make debugging possible, but I'm not sure if it needs another message specifically for a failure parsing the string.